Makefile.am

```diff
@@ -86,6 +85,7 @@ nobase_include_HEADERS = \
 	metal/led.h \
 	metal/lock.h \
 	metal/machine.h \
+	metal/machine-inline.h \
```

Comment:
Should we make it `metal/machine/inline.h` so that it matches `metal/machine/platform.h`? Or maybe change `platform.h` to match this one?